tests/contrib/hooks/test_slack_webhook_hook.py

```diff
@@ -91,6 +92,16 @@ def test_build_slack_message(self):
         # Then
         self.assertEqual(self.expected_message, message)
 
+    def test_get_base_url_default(self):
+        hook = SlackWebhookHook(http_conn_id='http_default')
+        hook.get_conn()
+
+        self.assertEqual(hook._get_base_url(), 'https://www.google.com/')
```

Comment:
This is obviously not useful for slack, and a test of the code only: but under what circumstances would you want to change the default URL?

Reply:
According to this article:https://stackoverflow.com/questions/52054427/how-to-integrate-apache-airflow-with-slack/52054663

SlackWebhookOperator could be set like this:
```
from airflow.contrib.operators.slack_webhook_operator import SlackWebhookOperator

slack_msg="Hi Wssup?"

slack_test =  SlackWebhookOperator(
    task_id='slack_test',
    http_conn_id='slack_connection',
    webhook_token='/1234/abcd',
    message=slack_msg,
    channel='#airflow_updates',
    username='airflow_'+os.environ['ENVIRONMENT'],
    icon_emoji=None,
    link_names=False,
    dag=dag)
```

And set the host of slack in connection like this: `host=https://hooks.slack.com/services/`

The argument-`http_conn_id` will be used for getting the hostname of Slack. But in current version, `http_conn_id` can't be passed the parent class-`HTTPHOOK`, so that `HTTPHook` will read the host by the default http_conn_id `http_default` and send the msg to `https://www.google.com/YOUR_ENDPOINT`.

Reply:
@ashb Does it make sense?

Reply:
I think the question is, do we ever want to set it to something different than `https://hooks.slack.com/services/`? I searched maybe for sandboxing, but couldn't find it that quickly.